docs/_posts/AbdullahMubeenAnwar/2025-07-25-phi_4_mini_instruct_q4_k_m_gguf_en.md

```
@@ -0,0 +1,126 @@
+])
+
+data = spark.createDataFrame([
+    ["The moon is "]
+]).toDF("text")
+
+model = pipeline.fit(data)
+result = model.transform(data)
+
+result.select("completions").show(truncate=False)
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)